Tests/Functional/SelectedTest/main.swift

```diff
@@ -26,19 +26,20 @@ class ExecutedTestCase: XCTestCase {
 // CHECK-TESTCASE: Test Case 'ExecutedTestCase.test_bar' passed \(\d+\.\d+ seconds\).
 // CHECK-ALL:      Test Case 'ExecutedTestCase.test_bar' started.
 // CHECK-ALL:      Test Case 'ExecutedTestCase.test_bar' passed \(\d+\.\d+ seconds\).
-    func test_bar() {}
+    func test_bar() {
+    }
```

Comment:
I also prefer the former syntax. I have made this change, as other methods in XCTest use this syntax. I could change those 'other' methods to this syntax though (which I would also prefer).